aplt/runner.py

```diff
@@ -54,11 +57,22 @@ def __init__(self, load_runner, websocket_url, statsd_client, scenario,
         # Processor and Websocket client vars
         self._scenario = scenario
         self._scenario_args = scenario_args
+        self._scenario_kw = scenario_kw
         self._processors = 0
         self._ws_clients = {}
         self._connect_waiters = deque()
         self._load_runner = load_runner
         self._stat_client = statsd_client
+        self._vapid = Vapid()
+        if "vapid_private_key" in self._scenario_kw:
+            self._vapid = Vapid(
+                private_key=self._scenario_kw.get(
+                    "vapid_private_key"))
+        else:
+            self._vapid.generate_keys()
+        self._claims = None
+        if "vapid_claims" in self._scenario_kw:
+            self._claims = self._scenario_kw.get("vapid_claims")
```

Comment:
You can send various claims by adding them to send_notification(), varying the keys takes a bit more effort since it's (in theory) and abnormal case. 

I did notice that in scenarios.py, I'm missing the vapid data block. That probably got lost in a merge. Will add that back in.